simplq/src/store/queues/queuesAsyncActions.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-param-reassign */
+import useAuth, { makeAuthedRequest } from 'api/auth';
+
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { QueueRequestFactory } from 'api/requestFactory';
+import * as RequestFactory from 'api/requestFactory';
```

Comment:
We will standardise here to one or the other?

Reply:
I am leaning towards one `RequestFactory`, as described [earlier](https://github.com/SimplQ/simplQ-frontend/pull/507/files#r569711806).